crates/builder/src/tasks/tx_poller.rs

```diff
@@ -42,7 +42,7 @@ impl TxPoller {
     /// unique transactions that haven't been seen before are sent into the builder pipeline.
     pub async fn check_tx_pool(&mut self) -> Result<Vec<TxEnvelope>, Error> {
         let mut unique: Vec<TxEnvelope> = Vec::new();
-        let result = self.client.get(self.config.tx_pool_url.to_string() + "/get").send().await?;
+        let result = self.client.get(self.config.tx_pool_url.to_string() + "get").send().await?;
```

Comment:
Was this actually causing an issue with the slash?

Reply:
this was what was causing the originally observed bug, yes